java/api/src/main/java/io/ray/api/utils/parallelactor/ParallelActor.java

```diff
@@ -0,0 +1,46 @@
+package io.ray.api.utils.parallelactor;
+
+import io.ray.api.ActorHandle;
+import io.ray.api.call.ActorTaskCaller;
+
+public class ParallelActor<A> implements ParallelActorCall<A> {
+
+  private ParallelStrategy strategy;
+  private int parallelNum;
+  private int lastIndex = -1;
```

Comment:
`-1` may not be a good initial value.

Reply:
Hum, what's your suggestion for that?

Reply:
I'd say `lastIndex` is only used for the round-robin strategy. The always-first strategy doesn't utilize this field. It perfectly makes sense if there's a class to describe the current state (different implementations based on strategy) and move the `lastIndex` to the round-robin implementation. This is also more extensible if we consider more potential strategies in the future. (Note that `ParallelActor` is in the API package.)

Reply:
Got your point.
I made the `Strategy` as an interface in API module, then `AlwaysFirstStrategy` and `RoundRobinStrategy` are in runtime module.